src/Sarif.Viewer.VisualStudio.UnitTests/ErrorList/SarifErrorListItemTests.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using FluentAssertions;
+using Microsoft.CodeAnalysis.Sarif;
+using Xunit;
+
+namespace Microsoft.Sarif.Viewer.VisualStudio.UnitTests
+{
+    public class SarifErrorListItemTests
+    {
+        [Fact]
+        public void SarifErrorListItem_LineMarker_Region_StartLine()
```

Comment:
>LineMarker_Region_StartLine [](start = 39, length = 27)

Our convention is to have test names describe what the test demonstrates. This name, in contrast, describes the input data ("Test what happens when a region has a start line").

I suggest:
```
SarifErrorListItem_WhenRegionHasStartLine_HasLineMarker
SarifErrorListItem_WhenRegionHasNoStartLine_HasNoLineMarker
SarifErrorListItem_WhenRegionIsAbsent_HasNoLineMarker
```

Reply:
Will change these.